src/common/store-data-to-scan-node-result-converter.ts

```diff
@@ -21,6 +21,7 @@ import { ScannerRuleInfo, ScannerRuleInfoMap } from 'scanner/scanner-rule-info';
 
 export type ScanNodeResult = UnifiedResult & {
     rule: UnifiedRule;
+    isSelected?: boolean;
```

Comment:
Correct me if I'm wrong, but I looked through the code and I don't think `isSelected` is referenced anywhere. It is needed for `CardResult` but not for `ScanNodeResult` and is set in `rule-based-view-model-provider.ts` `createCardResult`, which accepts a `ScanNodeResult`. I think we can remove it from the object returned in `convertAssessmentResultToScanNodeResult` below.

Reply:
Agreed and removed isSelected property from function convertAssessmentResultToScanNodeResult